heidrun/mdl_map.rb

```diff
@@ -0,0 +1,77 @@
+Krikri::Mapper.define(:mdl_map, :parser => Krikri::JsonParser) do
```

Comment:
Would it be useful to pass a parser argument to declare `record` as the root node? This would allow us to reduce the redundancy in the mapping.